src/lib/viewers/doc/DocPreloader.js

```diff
@@ -268,11 +268,19 @@ class DocPreloader extends EventEmitter {
      * @return {void}
      */
     resize() {
-        if (!this.pdfData || !this.preloadEl) {
+        if (!this.preloadEl || (!this.pdfData && !this.imageEl)) {
```

Comment:
This was causing the resize issue with the preview sidebar. If an image didn't have exif data burnt into it, we wouldn't be able to resize and would see that overlap.